projects/js-packages/components/tools/jp-redirect/index.ts

```diff
@@ -38,7 +38,7 @@ export default function getRedirectUrl( source: string, args: GetRedirectUrlArgs
 		queryVars.source = encodeURIComponent( source );
 	}
 
-	Object.keys( args ).map( argName => {
+	Object.keys( args ).forEach( argName => {
```

Comment:
There's no need for `map` as it will create new array, while we don't need it.

Reply:
You could go a step further and turn it into `for ( const argName of Object.keys( args ) ) {`.